easybuild/easyconfigs/r/Rmath/Rmath-3.3.1-intel-2016b.eb

```diff
@@ -0,0 +1,49 @@
+# Easyconfig for Rmath
+# Author: Caspar van Leeuwen
+# SURFsara, Amsterdam, The Netherlands
+
+easyblock = 'ConfigureMake'
+
+name = 'Rmath'
+version = '3.3.1'
+
+homepage = "https://www.r-project.org/"
+description = """Rmath is the standalone version of the R math library.
+ Rmath can be used in your own C/C++ routines."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = ['R-%(version)s.tar.gz']
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s/']
+checksums = ['f50a659738b73036e2f5635adbd229c5']
+
+# Copied from R-3.3.1-intel-2016b.eb.
+# Not sure if BLAS/LAPACK are needed/used when compiling R-math, R documentation doesn't specify this
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
```

Comment:
the name suggests it's used, and you should be able to tell from the build log?

Reply:
I would have guessed a math library uses it... but the build log doesn't show any -l... linking to libRmath.so that suggests the use of BLAS or LAPACK. In fact, an ldd on libRmath.so shows it is not linked to either. I'll remove it from the EasyConfig.